src/test/java/lv/ctco/javaschool/vote/boundary/VoteApiTest.java

```diff
@@ -194,6 +195,30 @@ void submitDailyVote_Unhappy_with_Comment() {
         verify(em, times(1)).persist(any(DailyVote.class));
     }
 
+    @Test
+    @DisplayName("GetAllEventVotesTest: Checks if method returns correct dto value")
+    void getAllEventVotesTest(){
+        List<EventVote> eventVoteList= new ArrayList<>();
+        EventVote ev = new EventVote();
+        Event event = new Event();
+
+        event.setEventName("testEvent");
+        ev.setComment("test");
+        ev.setMood(MoodStatus.HAPPY);
+        ev.setUser(user1);
+        ev.setEvent(event);
+        eventVoteList.add(ev);
+
+        when(voteStore.getAllEventVotes())
+                .thenReturn(eventVoteList);
+
+        List<EventVoteDto> test = voteApi.getAllEventVotes();
```

Comment:
Add a check that this list contains exactly one vote (should not return any extra items).